vm/src/builtins/bytes.rs

```diff
@@ -606,6 +607,21 @@ impl AsSequence for PyBytes {
     };
 }
 
+impl AsNumber for PyBytes {
+    const AS_NUMBER: PyNumberMethods = PyNumberMethods {
+        remainder: Some(|number, other, vm| {
+            let formatted = number
+                .obj
+                .downcast_ref::<PyBytes>()
+                .unwrap()
+                .inner
+                .cformat(other.to_owned(), vm)?;
+            Ok(formatted.to_pyobject(vm))
```

Comment:
```suggestion
            Ok(Self::number_downcast(number)
                .inner
                .cformat(other.to_pyobject(vm), vm)?
                .to_pyobject(vm))
```

Reply:
`number_downcast` will do what we want 😊

Reply:
@Snowapril Thanks for recognizing to me 😄